testem.js

```diff
@@ -13,7 +13,6 @@ module.exports = {
         // --no-sandbox is needed when running Chrome inside a container
         process.env.CI ? '--no-sandbox' : null,
         '--headless',
-        '--disable-gpu',
```

Comment:
See https://github.com/ember-cli/ember-cli/pull/8774 for why this change was made. The removal is part of the blueprints now.